Sources/Apollo/ApolloStore.swift

```diff
@@ -141,17 +141,17 @@ public final class ApolloStore {
       self.cacheKeyForObject = cacheKeyForObject
     }
     
-    public func readObject<Query: GraphQLQuery>(forQuery query: Query) throws -> JSONObject {
-      return try readObject(forSelectionSet: Query.selectionSet, withKey: rootKey(forOperation: query), variables: query.variables)
+    public func read<Query: GraphQLQuery>(query: Query) throws -> Query.Data {
```

Comment:
Oh, sorry, totally missed your reponse. But the answer is that `readFragment` doesn't take a fragments as an argument, but a fragment type. So the signature is `readFragment(ofType:)`.